### ISSUE TITLE
Problem with installation

### ISSUE THREAD

**Issue by KaranKapur123** [minerllabs/minerl#345]
while running the command pip3 install --upgrade minerl
i got the following output
C:\Users\Karan>pip3 install --upgrade minerl
Collecting minerl
  Using cached minerl-0.3.5.tar.gz (36.3 MB)
Requirement already satisfied, skipping upgrade: gym>=0.12.1 in c:\users\karan\appdata\local\programs\python\python38\lib\site-packages (from minerl) (0.17.2)
Collecting opencv-python>=4.1.0.25
  Using cached opencv_python-4.3.0.36-cp38-cp38-win_amd64.whl (33.4 MB)
Requirement already satisfied, skipping upgrade: setuptools>=40.6.2 in c:\users\karan\appdata\local\programs\python\python38\lib\site-packages (from minerl) (49.1.0)
Collecting tqdm>=4.32.2
  Using cached tqdm-4.47.0-py2.py3-none-any.whl (66 kB)
Requirement already satisfied, skipping upgrade: numpy>=1.16.2 in c:\users\karan\appdata\local\programs\python\python38\lib\site-packages (from minerl) (1.19.0)
Requirement already satisfied, skipping upgrade: requests>=2.20.0 in c:\users\karan\appdata\local\programs\python\python38\lib\site-packages (from minerl) (2.24.0)
Collecting ipython>=7.5.0
  Using cached ipython-7.16.1-py3-none-any.whl (785 kB)
Collecting typing>=3.6.6
  Using cached typing-3.7.4.1-py3-none-any.whl (25 kB)
Collecting lxml>=4.3.3
  Using cached lxml-4.5.1-cp38-cp38-win_amd64.whl (3.5 MB)
Collecting psutil>=5.6.2
  Using cached psutil-5.7.0-cp38-cp38-win_amd64.whl (235 kB)
Collecting Pyro4>=4.76
  Using cached Pyro4-4.80-py2.py3-none-any.whl (89 kB)
Collecting coloredlogs>=10.0
  Using cached coloredlogs-14.0-py2.py3-none-any.whl (43 kB)
Collecting matplotlib==3.0.3
  Using cached matplotlib-3.0.3.tar.gz (36.6 MB)
    ERROR: Command errored out with exit status 1:
     command: 'c:\users\karan\appdata\local\programs\python\python38\python.exe' -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'C:\\Users\\Karan\\AppData\\Local\\Temp\\pip-install-7vwmt8g7\\matplotlib\\setup.py'"'"'; __file__='"'"'C:\\Users\\Karan\\AppData\\Local\\Temp\\pip-install-7vwmt8g7\\matplotlib\\setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base 'C:\Users\Karan\AppData\Local\Temp\pip-pip-egg-info-9rj2neqd'
         cwd: C:\Users\Karan\AppData\Local\Temp\pip-install-7vwmt8g7\matplotlib\
    Complete output (47 lines):
    ============================================================================
    Edit setup.cfg to change the build options

    BUILDING MATPLOTLIB
                matplotlib: yes [3.0.3]
                    python: yes [3.8.2 (tags/v3.8.2:7b3ab59, Feb 25 2020,
                            23:03:10) [MSC v.1916 64 bit (AMD64)]]
                  platform: yes [win32]

    REQUIRED DEPENDENCIES AND EXTENSIONS
                     numpy: yes [version 1.19.0]
          install_requires: yes [handled by setuptools]
                    libagg: yes [pkg-config information for 'libagg' could not
                            be found. Using local copy.]
                  freetype: no  [The C/C++ header for freetype
                            (freetype2\ft2build.h) could not be found.  You may
                            need to install the development package.]
                       png: no  [The C/C++ header for png (png.h) could not be
                            found.  You may need to install the development
                            package.]
                     qhull: yes [pkg-config information for 'libqhull' could not
                            be found. Using local copy.]

    OPTIONAL SUBPACKAGES
               sample_data: yes [installing]
                  toolkits: yes [installing]
                     tests: no  [skipping due to configuration]
            toolkits_tests: no  [skipping due to configuration]

    OPTIONAL BACKEND EXTENSIONS
                       agg: yes [installing]
                     tkagg: yes [installing; run-time loading from Python Tcl /
                            Tk]
                    macosx: no  [Mac OS-X only]
                 windowing: yes [installing]

    OPTIONAL PACKAGE DATA
                      dlls: no  [skipping due to configuration]

    ============================================================================
                            * The following required packages can not be built:
                            * freetype, png
                            * Please check http://gnuwin32.sourceforge.net/packa
                            * ges/freetype.htm for instructions to install
                            * freetype
                            * Please check http://gnuwin32.sourceforge.net/packa
                            * ges/libpng.htm for instructions to install png
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

Please help

**Comment by CalebThePerson** [comment]
yes I get the same error please help, I follow all of the instructions correctly but it keeps giving me the same error.  I even reinstalled the packages before hand and I am still getting the same errors.

**Comment by simon1506** [comment]
I got the same error when i tried installing minerl with the newest Python version (3.8.5). Using Python 3.7.8 fixed it for me.
I think the error occures because minerl installs an older version of matplot which doesn't support the newest Python version 3.8.5.
